arviz/plots/khatplot.py

```diff
@@ -116,6 +121,22 @@ def plot_khat(
         >>> az.plot_khat(loo_radon, color=colors)
 
     """
+    if hover_label and mpl.get_backend() not in mpl.rcsetup.interactive_bk:
+        try:
+            from IPython import get_ipython
+            ipython = get_ipython()
+            ipython.magic("matplotlib")
+        except (ImportError, AttributeError):
+            hover_label = False
+            warnings.warn(
+                "hover labels are only available with interactive backends. To switch to an "
+                "interactive backend from ipython or jupyter, use `%matplotlib` there is no need "
```

Comment:
I don't this works on all backends

Reply:
I am not sure either, but so far it has worked for me locally in many different configurations. Maybe it would be better to directly point to matplotlib and ipython docs on matplotlib backends?

Reply:
I can also remove the `try` to block to change to an interactive backend and force the user to change it manually always.